core/modules/core/src/com/haulmont/yarg/formatters/impl/DocxFormatter.java

```diff
@@ -117,6 +135,21 @@ protected void updateTableOfContents() {
         }
     }
 
+    @SuppressWarnings("rawtypes,unchecked")
+    protected void replaceNewLines(JaxbXmlPart documentPart) {
```

Comment:
Why update an entire document? 
Substitution of values from bands occurs only in the `formatValue()` method. Thus, it can be assumed that a change in the method `formatValue()` will be sufficient.

Reply:
Unfortunately that doesn't work, because the value retrieved from `formatValue()` is set into a `org.docx4j.wml.Text` object later on in the processing. E.g. see `com.haulmont.yarg.formatters.impl.docx.TableManager#fillRowFromBand` or `com.haulmont.yarg.formatters.impl.docx.TextWrapper#fillTextWithBandData` 

The replacement from `NEWLINE_REPLACE_CHARACTER` to `"</w:t><w:br/><w:t>"` will work, but the replaced string will be escaped and rendered as `</w:t><w:br/><w:t>` into the DOCX document